ui-kit/src/main/kotlin/org/p2p/uikit/organisms/UiKitSectionHeader.kt

```diff
@@ -0,0 +1,52 @@
+package org.p2p.uikit.organisms
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import androidx.core.view.isVisible
+import org.p2p.uikit.R
+import org.p2p.uikit.databinding.WidgetSectionHeaderBinding
+
+class UiKitSectionHeader @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : LinearLayout(context, attrs, defStyleAttr) {
+
+    private val binding = WidgetSectionHeaderBinding.inflate(LayoutInflater.from(context), this)
```

Comment:
```suggestion
    private val binding = inflateViewBinding<WidgetSectionHeaderBinding>()
```

Reply:
We should use extensions after moving them to utils module

Reply:
sry, i totally forgot about that